### PR TITLE
#9Task. Minko Stas

### PR DESCRIPTION
**Pull request by StanislavM31**
https://travis-ci.com/github/StanislavM31/js-assignments/builds/209188826

### CODE REVIEW

**Comment by StanislavM31** [comment]
@vramaniuk
 maybe u can check?

**Comment by IlyaMokin** [comment]
the PR has redundant changes in styling, spaces, comments.... I also see that some merge conflicts were not solved
Please follow the rules:
 - do not add any non logic changes to code if review is required
 - if the changes in styling are important - you can do it as separate PR with styling only changes to avoid noise in the main PR

File: task/01-strings-tasks_BACKUP_894.js [hunk 1]
```diff
@@ -0,0 +1,365 @@
...
+ * @return {string}
+ *
+ * @example
+ *   'aa', 'bb' => 'aabb'
+ *   'aa',''    => 'aa'
+ *   '',  'bb'  => 'bb'
+ */
+function concatenateStrings(value1, value2) {
+<<<<<<< HEAD
+    throw new Error('Not implemented');
```
Comment (ilya4321) [inline task/01-strings-tasks_BACKUP_894.js:26]: merge conflict?

File: task/01-strings-tasks_BACKUP_894.js [hunk 2]
```diff
@@ -0,0 +1,365 @@
...
+=======
+    return value.length;
+>>>>>>> 902a6c9... check #1 task1-4
+}
+
+/**
+ * Returns the result of string template and given parameters firstName and lastName.
+ * Please do not use concatenation, use template string :
+ * https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/template_strings
```
Comment (ilya4321) [inline task/01-strings-tasks_BACKUP_894.js:55]: changes in comments

**Pull request closed without merging**